code/modules/admin/team_panel.dm

```diff
@@ -147,11 +147,12 @@
 	//qdel maybe
 
 /datum/team/proc/admin_add_member(mob/user)
-	var/list/minds = list()
-	for(var/mob/M in GLOB.mob_list)
-		if(M.mind)
-			minds |= M.mind
-	var/datum/mind/value = input("Select new member:", "New team member", null) as null|anything in sortNames(minds)
+	var/list/candidates = list()
+	for(var/mob/M in GLOB.player_list)
+		if(M?.mind.special_role)
+			continue
+		candidates |= M.mind
```

Comment:
Could probably just use += here, I don't see why there would be multiple instances of the same mob in player list

Reply:
```suggestion
		candidates += M.mind
```